webapp/src/ts/effects/global.effects.ts

```diff
@@ -72,10 +72,10 @@ export class GlobalEffects {
       .catch(() => false);
   }
 
-  private navigate(nextUrl, cancelCallback) {
+  private navigate(nextUrl: string, cancelCallback: Function) {
```

Comment:
```suggestion
  private navigate(nextUrl: string, cancelCallback: () => void) {
```

That would be great if we can type the function a bit more strongly here  
I initially wanted to use typescript's builtin equivalent `VoidFunction` but eslint complains about it

Reply:
Merci! I've applied the suggestion